src/vite/utils/tailwindSafelist.js

```diff
@@ -29,6 +33,16 @@ let vuelessConfig = {};
   }
 })();
 
+export const { cx } = defineConfig({
+  hooks: {
+    onComplete: (classNames) => twMerge(classNames).replace(NESTED_COMPONENT_REG_EXP, ""),
+  },
+});
```

Comment:
Why you need to redefine cva again here?

Where is  `twMerge` imported?

Reply:
Its created above and this is the reason why we have to redefine cva again. twMerge in vueless uses vuelessConfig imported with glob, this is one uses vueless imported with dynamic import